ts/data.ts

```diff
@@ -1,6 +1,36 @@
+interface Entry {
+  entryID: number;
+  title: string;
+  imgUrl: string;
+  notes: string;
+}
 const data = {
   view: 'entry-form',
-  entries: [],
+  entries: [] as Entry[],
   editing: null,
   nextEntryId: 1,
 };
+
```

Comment:
This object should be getting created by calling `loadData`. `const data = loadData()`